code/game/jobs/jobs.dm

```diff
@@ -35,6 +35,7 @@ var/const/BOTANIST			=(1<<2)
 var/const/CHEF				=(1<<3)
 var/const/JANITOR			=(1<<4)
 var/const/LIBRARIAN			=(1<<5)
+var/const/ENTERTAINER		=(1<<5)
```

Comment:
Change it to 
`=(1<<15)`
the 5 is already used by librarian, and that results in the issue with the job pref